ONSdigital/census-int-common-service
# Cr 450 relax timestamp formatting rules

# Motivation and Context
Motivation and Context
Make timestamp parsing accept timestamps which may not include milliseconds.

This change is trickier than it sounds as we require a lot of flexibility on the way in to our services but a fixed and known format on the way out.

# What has changed
Default object mapper has been updated. This is intended to be used by CC, RH and Event Generator.

# How to test?
Manually send in a UAC/CASE with a header timestamp which is missing the milliseconds field. See README.md in RH service.

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-450
